### PR TITLE
fix: 397672 - Catch 409 Conflict on Change Dog Status

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/routes/cdo/edit/change-status.js [hunk 1]
```diff
@@ -4,8 +4,30 @@ const getUser = require('../../../auth/get-user')
 const ViewModel = require('../../../models/cdo/edit/change-status')
 const { getCdo } = require('../../../api/ddi-index-api/cdo')
 const { updateStatus } = require('../../../api/ddi-index-api/dog')
-const { validatePayload } = require('../../../schema/portal/edit/change-status')
+const { validatePayload, duplicateMicrochipSchema } = require('../../../schema/portal/edit/change-status')
 const { addBackNavigation, addBackNavigationForErrorCondition } = require('../../../lib/back-helpers')
+const { ApiConflictError } = require('../../../errors/api-conflict-error')
+
+const changeStatusPostFailAction = async (request, h, error) => {
```
Comment (jbarnsley10) [inline app/routes/cdo/edit/change-status.js:11]: I didn't think there was a particular need to extract this out to a method (unless you needed it for unit testing)??

File: app/schema/portal/edit/change-status.js [hunk 2]
```diff
@@ -7,6 +7,13 @@ const changeStatusSchema = Joi.object({
   })
 }).required()
 
+const duplicateMicrochipSchema = Joi.object({
+  indexNumber: Joi.string().forbidden().messages({
+    '*': 'The microchip number is in use on another record.'
```
Comment (jbarnsley10) [inline app/schema/portal/edit/change-status.js:12]: I think the wording is due to change here

File: app/routes/cdo/edit/change-status.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/routes/cdo/edit/change-status.js:11]: it's being used in the POST handler - so was wanting to make sure it's not repeating

File: app/schema/portal/edit/change-status.js (hunk 2 above)
Comment (whitewaterdesign) [reply app/schema/portal/edit/change-status.js:12]: this is what was communicated in the meeting.

**Review by jbarnsley10 — APPROVED** [review]

**Pull request merged**